scripts/create_sample_worksheet.py

```diff
@@ -145,6 +145,8 @@ def _create_dependencies(self):
                     'run',
                     '--request-memory',
                     '10m',
+                    '--request-docker-image',
+                    'python:3.6.10-slim-buster',
```

Comment:
This has some repetition -- we should refactor this to use something similar to `_run_command` in a separate PR.